### ISSUE TITLE
Bottom of page is hidden under footer (was: Submit button is missing) in responsive / mobile view

### ISSUE THREAD

**Issue by annirytkonen** [apluslms/a-plus#1021]
There is no submit button in student's mobile /responsive view, at least when answering multiple-choice questions in Chrome.

![Screenshot_20220418-155112_Chrome](https://user-images.githubusercontent.com/59993690/164019086-75be3555-e940-4c5c-8cd5-8db13de6284d.jpg)

**Comment by annirytkonen** [comment]
there's another comment via the feedback form: 
"Puhelimella tehtäviä, jotka ovat alimpana sivulla, ei pääse palauttamaan, koska sivua ei pääse rullaamaan tarpeeksi alaspäin. Sama tapahtuu myös tekstien kanssa; niitä ei saa luettua loppuun puhelimen selaimella (kokeilin sekä Googlea että Safaria), koska sivua ei saa rullattua tarpeeksi alaspäin."

--> it seems this is not only the submit button but the bottom of the page overall is hidden below the footer. Improvement suggestions:

"A) Palautus-nappi myös tehtävälaatikon yläpäähän varmuuden vuoksi.

B) Jokaisen sivun loppuun hieman tyhjää tms. jotain tekstiä (alatunniste), että mikään rivi ei leikkaannu pois."

**Comment by Mankro** [comment]
This might be already fixed in the current version. The screenshot in the first post shows the older version number v1.11 (October 2021).

If the problem is caused by the padding in the site footer, then increasing the padding is easy. I have increased the padding in the past when I added new items to the footer.

To be more precise, the footer itself doesn't have padding now, but the site main content has padding-bottom so that the main content does not overlap with the footer. In the current production version, I can't see any overlap on narrow screens. There is probably nothing left to fix.

https://github.com/apluslms/a-plus/blob/e832ba304b22392e8acc0e47292a4ccd3dae24b0/assets/sass/layout/_base.scss#L10-L15

> A) Palautus-nappi myös tehtävälaatikon yläpäähän varmuuden vuoksi.
> extra submit button at the top of the assignment

This is not really part of this issue. This issue is about fixing potential issues in the site footer styles (padding or such). It is a very different matter to modify the assignment submission forms and I wouldn't go there now.

**Comment by annirytkonen** [comment]
yet another request from the feedback form & Tampere version 1.11.

**Comment by ihalaij1** [comment]
I looked into this and it seems that this happens only in PLUSSA, but not on A+ (even with A+ v1.11).
Therefore, this could be an issue with PLUSSA's custom theme.

**Comment by annirytkonen** [comment]
ok, thanks, I'll inform Jyke and close this issue!
